parkings/tests/api/operator/test_permit.py

```diff
@@ -210,3 +217,68 @@ def test_operator_and_enforcers_cannot_see_each_others_permit(
     assert set([permit['id'] for permit in json_response['results']]) == set(
         [permit.id for permit in enforcer_permit_list]
     )
+
+
+def test_operator_can_only_create_permit_on_permitareas_where_she_is_permitted(
```

Comment:
These test names could be changed in similar manner that was done in e62b6cd.

Reply:
Will take care of this in another PR.